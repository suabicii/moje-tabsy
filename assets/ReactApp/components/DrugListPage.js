import React from "react";
import DrugList from "./DrugList";

function DrugListPage() {
    return (
        <>
            <h1 className="text-center mt-5 mt-md-0">Moje leki i suplementy</h1>
            <DrugList isEditMode={true}/>
        </>
    );
}

export default DrugListPage;