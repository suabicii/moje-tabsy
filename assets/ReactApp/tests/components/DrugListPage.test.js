import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import DrugListPage from "../../components/DrugListPage";

it('should correctly render DrugListPage', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<DrugListPage/>);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
}); 