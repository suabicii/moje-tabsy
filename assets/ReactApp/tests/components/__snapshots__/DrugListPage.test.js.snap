// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render DrugListPage 1`] = `
<React.Fragment>
  <h1
    className="text-center mt-5 mt-md-0"
  >
    Moje leki i suplementy
  </h1>
  <DrugList
    isEditMode={true}
  />
</React.Fragment>
`;
